types/mapcmd/map.go

```diff
@@ -74,7 +74,7 @@ func getMap(ctx context.Context, clientConfig *hazelcast.Config, mapName string)
 	return
 }
 
-const MapUseExample = "map use m1          # sets the default map name to m1 unless set explicitly"
+const MapUseExample = "map use m1    # sets the default map name to m1 unless set explicitly"
```

Comment:
Why is this change necessary?

Reply:
On a previous PR we decided to remove tabs and use spaces